src/components/dashboard/sidebar/SubsidiesSummary.jsx

```diff
@@ -85,6 +93,25 @@ const SubsidiesSummary = ({
   const hasAvailableSubsidyOrRequests = (
     hasActiveLicenseOrLicenseRequest || hasAssignedCodesOrCodeRequests || learnerCreditSummaryCardData
   );
+  const [assignmentOnlyLearner, setAssignmentOnlyLearner] = useState(false);
+  useEffect(() => {
+    const data = redeemableLearnerCreditPolicies?.flatMap(item => item?.learnerContentAssignments || []);
+    const assignmentsData = sortAssignmentsByAssignmentStatus(data);
```

Comment:
[nit/question]: do we need to sort assignments before filtering for active assignments, since we're not displaying assignments within this component?